brainframe/cli/main.py

```diff
@@ -18,10 +26,14 @@ def main():
         "command", default=None, nargs="?", help=i18n.t("portal.command-help")
     )
 
+    config.load()
+
     # This environment variable must be set as it is used by the
     # docker-compose.yml to find the data path to volume mount
-    if not env_vars.data_path.is_set():
-        os.environ[env_vars.data_path.name] = str(env_vars.data_path.default)
+    if config.data_path.name not in os.environ:
+        os.environ[config.data_path.env_var_name] = str(
+            config.data_path.default
+        )
```

Comment:
Shouldn't you be checking if `config.data_path.env_var_name` is in the path?

Reply:
Aw frick!